core/storage/suggestion/gae_models.py

```diff
@@ -479,17 +480,29 @@ def get_in_review_translation_suggestions(
                 As a user cannot review their own suggestions, suggestions
                 authored by the user will be excluded.
             language_codes: list(str). The list of language codes.
+            exp_ids: list(str). Exploration IDs matching the target ID of the
```

Comment:
Should document the behavior of an empty list here as well.

Reply:
Done